tee-worker/ts-tests/common/utils/context.ts

```diff
@@ -12,7 +12,8 @@ import { IntegrationTestContext, teeTypes, EnclaveResult, Web3Wallets } from '..
 const crypto = require('crypto');
 
 // maximum milliseconds that we wait in listening events before we timeout
-const listenTimeoutInMilliSeconds = 3 * 60 * 1000;
+// set to 5 minutes as identity-hub to ensure all tests have enough time to get events
```

Comment:
Do we need to increase the timeout here? I feel like it may not be necessary to extend the time. Normally, we expect to get the result within 15 blocks (`bulk_identity/vc` may vary depending on the situation). So, what I mean is, I'm not sure of the significance of increasing the time here. If it is necessary, please let me know. Thank you. :)

Reply:
I agree with @Verin1005 

I would ask if we can go a little bit further and have exact block number expectations for invocation to be finished.
It would help us detect situations when changes causes invocations to take longer than expected.

For example we have CreateIdentity call, and we expect this call to be finished within 3 blocks.

Reply:
Used 15 block number for now, removed the timeout completely.